README.md

```diff
@@ -54,6 +54,7 @@ An ([IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)) is
 - [Atheos](https://atheos.io) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/Atheos/Atheos) [<img title="Self Hosted" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/docker.svg" />](https://hub.docker.com/r/hlsiira/atheos) - Run most languages using a PHP/Browser based IDE with a terminal, git integration, and syntax highlighting.
 - [HASH](https://hash.ai) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Build and run JavaScript, Python, or Rust multi-agent simulations in the browser. Provides visualization, analysis, and experiment features.
 - [goormIDE](https://ide.goorm.io) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run almost every programming languages with real-time collaboration, live pair programming, and built-in messenger. 
+- [Coding Rooms](https://codingrooms.com/compiler/) - Edit and run JavaScript, Java, Python, C++ and more in a collaborative environment with intellisense, live chat, audio and video conferencing, and real-time editing. Each workspace offers an interactive terminal and containerized environment.
```

Comment:
I tried JS and it does have intellisense, but then I tried Java and it does not. I don't think this belongs in the Full IDE category

![image](https://user-images.githubusercontent.com/229881/127057925-57a5493f-2ca3-4bbd-b547-80da6f5e604e.png)

Reply:
I hear your concern with this. Intellisense is only supported for non-web langauges using the browser-based VS Code support on Coding Rooms. Here's an example (please fork the workspace to edit and try out Intellisense): https://app.codingrooms.com/w/sz7bd0rv7hQs

Reply:
I still have the same problem with Java after clicking your link and then clicking the "Fork" button

Reply:
I see... I just recorded a screencast of Java intellisense using the link I shared above. Please let me know if this helps and if there is some issue causing it to fail to load on your end. Thank you!

https://user-images.githubusercontent.com/10859025/127060930-ed639a40-de3b-4daa-931d-4a04537524be.mp4

Reply:
```suggestion
- [Coding Rooms](https://codingrooms.com/compiler/) - Edit and run JavaScript, Java, Python, C++ and more in a collaborative environment with intellisense, live chat, audio and video conferencing, and real-time editing, and terminal.
```